### ISSUE TITLE
OAuth token does not get refreshed when they expire

### ISSUE THREAD

**Issue by jpd4emis** [PagerDuty/backstage-plugin-backend#87]
**Describe the bug**
OAuth token does not get refreshed when they expire after the day TTL it has.

**To Reproduce**
1. Configure OAuth token access
2. check that it works
3. wait a day
4. check that it works

**Expected behavior**
The token should get refreshed after a day and the API access should keep working

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - N/A

**Additional context**

**Comment by graemechristie** [comment]
@jpd4emis is the fix for this issue due to be merged/released any time soon ? We are currently restarting all of our backstage instances every day entirely due to this single issue.

**Comment by jpd4emis** [comment]
@graemechristie no, I think pagerduty have walked away from this when their developer relations left.

I had a support ticket in with this for months and have just been running a local copy in the plugin folder until they respond with something meaningful.

**Comment by martina-equinix** [comment]
@jpd4emis @graemechristie  The [fix](https://github.com/PagerDuty/backstage-plugin-backend/pull/106) for this was released in [0.9.3](https://github.com/PagerDuty/backstage-plugin-backend/releases/tag/v0.9.3) 🎉

**Comment by cdmicacc** [comment]
Hey folks! I’m happy to provide an update on this issue - it’s been fixed (by [this PR](https://github.com/PagerDuty/backstage-plugin-backend/pull/106)) and a new version of the plugin has been published! We appreciate your patience on this.

I’m also happy to announce that we will continue to maintain this plugin and will review any issues periodically to ensure the plugin remains up-to-date and functioning.

We’re always eager to hear our community’s feedback while using the plugin! Please join our [community forums](https://community.pagerduty.com/) for another venue to share feedback and chat with others in the community.
